go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -5709,9 +5723,22 @@ var DoltCherryPickTests = []queries.ScriptTest{
 				Expected: []sql.Row{{1, "one"}},
 			},
 			{
+				// An ignored table should still be present (and unstaged) after aborting the merge.
 				Query:    "select * from dolt_status;",
-				Expected: []sql.Row{},
+				Expected: []sql.Row{{"generated_foo", false, "new table"}},
```

Comment:
Maybe beyond the scope of this bug fix, but I would expect that an ignored table doesn't appear in `dolt_status`, similar to how ignored files don't appear in `git status`.

Reply:
I think when `dolt_ignore` was being added, the decision was to add a `ignored` column to the status table.

Either way, I agree that something should be done, and also agree that this is beyond the scope of the bug fix.